lib/client/workflow-client/workflowClient.js

```diff
@@ -155,7 +155,7 @@ WorkflowMediatorService.prototype.checkStatus = function checkStatus(workorder,
     status = CONSTANTS.STATUS.COMPLETE_DISPLAY;
   } else if (!workorder.assignee) {
     status = CONSTANTS.STATUS.UNASSIGNED_DISPLAY;
-  } else if (stepReview.nextStepIndex < 0) {
+  } else if (!result) {
```

Comment:
Thanks for response. Something for later: Can we change this to set explicit status on workflow as we transition, click on begin etc.?